docs/source/contents/4_draft_mapping.rst

```diff
@@ -67,6 +79,20 @@ Mapping
 identifier
 ==========
 
++------------------------+------------+-------------------+---------------------------------------------------------------------------------------+
+| Predicate              | Value Type | Range (if needed) | Usage Notes                                                                           |
++------------------------+------------+-------------------+---------------------------------------------------------------------------------------+
+| dbo:isbn               | Literal    |                   | Use for identifiers with type=""isbn                                                  |
```

Comment:
There's a little typo here `type=""isbin`

Reply:
Thanks Bridger for your eagle eyes - mine apparently are failing me.